### DISCUSSION TITLE
Copy English source strings when creating new translations

### DISCUSSION THREAD

**Discussion by sr258** [WeblateOrg/weblate#5599] [Q&A] [answered]
By default, all strings in a new translation created by Weblate are empty. The application for which I run a localization site expects untranslated string to be copies of the original English source strings. Is there a way of configuring Weblate to change how it creates new translations?

I also now have a number of translations that contain these empty strings. Can I somehow batch edit those to copy the English source strings for cases in which the translated string is empty?

**Comment by nijel** [comment, marked as answer]
I don't think there currently is such option. The general issue with this approach is that there is typically no way to distinguish non-translated string from a string intentionally kept the same while translating.

**Reply by sr258** [reply]
I agree that it's not a good approach, but that's how the application expects it and I can't change it, unfortunately.

**Reply by nijel** [reply]
Yes, it might have some use-cases, I've created an issue to track this feature: https://github.com/WeblateOrg/weblate/issues/5692
